docs/snippets/solid/storybook-preview-global-decorator.js.mdx

```diff
@@ -0,0 +1,11 @@
+```js
+// .storybook/preview.js
```

Comment:
@webblocksapp, to prevent the snippets from reaching a point they can become hard to maintain, could you apply the same pattern used for React [here](https://github.com/storybookjs/storybook/blob/next/docs/snippets/react/storybook-preview-global-decorator.js.mdx) and condense the decorator's snippets into a single one for JavaScript and one for TypeScript and remove the extra ones. As you already applied this to some extent but forgot to update the remainder examples.

Reply:
Fixed.